robyn/__init__.py

```diff
@@ -211,11 +222,14 @@ def start(self, host: str = "127.0.0.1", port: int = 8080, _check_port: bool = T
         port = int(os.getenv("ROBYN_PORT", port))
         open_browser = bool(os.getenv("ROBYN_BROWSER_OPEN", self.config.open_browser))
 
+        valid_socket, _socket = self.get_socket(host, port)
```

Comment:
`is_connection_valid, socket = self.check_socket_connection`

Reply:
So, a rule of thumb for naming conventions that I follow - the line should read like a sentence.